### PR TITLE
Idenitfy Graphics new sample

### PR DESCRIPTION
**Pull request by shokhinaToj**
Creates a new sample that identifies the graphic that is being tapped on.

### CODE REVIEW

File: assets/generated_samples_list.json [hunk 1]
```diff
@@ -902,6 +902,32 @@
     "title": "Generate offline map",
     "key": "generate_offline_map"
   },
+  "identify_graphics": {
+    "category": "Display Information",
+    "description": "Display an alert message when a graphic is clicked.",
```
Comment (jenmerritt) [inline assets/generated_samples_list.json:907]:
```suggestion
    "description": "Display an alert message when a graphic is tapped.",
```

File: lib/samples/identify_graphics/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+1. Create a `GraphicsOverlay` and add it to the `MapView`.
+2. Add a `Graphic` along with a `SimpleFillSymbol` to the graphics overlay.
+3. Identify the location clicked on the map view by the user with a `onTap` method.
+4. Identify the graphic on the map view with `identifyGraphicsOverlay(graphicsOverlay, screenPoint, tolerance, maximumResults)`.
+
+## Relevant API
+
+* Graphic
+* GraphicsOverlay
+* MapView
```
Comment (jenmerritt) [inline lib/samples/identify_graphics/README.md:26]:
```suggestion
* ArcGISMapView
```

File: lib/samples/identify_graphics/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+A user may wish to select a graphic on a map to view relevant information about it.
+
+## How to use the sample
+
+Tap on a graphic to identify it. You will see an alert dialog displayed.
+
+## How it works
+
+1. Create a `GraphicsOverlay` and add it to the `MapView`.
```
Comment (jenmerritt) [inline lib/samples/identify_graphics/README.md:17]:
```suggestion
1. Create a `GraphicsOverlay` and add it to the `ArcGISMapView`.
```

File: lib/samples/identify_graphics/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Identify graphics
+
+Display an alert message when a graphic is clicked.
```
Comment (jenmerritt) [inline lib/samples/identify_graphics/README.md:3]:
```suggestion
Display an alert message when a graphic is tapped.
```

File: assets/generated_samples_list.json (hunk 1 above)
Comment (jenmerritt) [reply assets/generated_samples_list.json:907]: This will be picked up when you update the readme

File: lib/models/samples_widget_list.dart [hunk 5]
```diff
@@ -42,64 +43,58 @@ import 'package:arcgis_maps_sdk_flutter_samples/samples/show_portal_user_info/sh
 import 'package:arcgis_maps_sdk_flutter_samples/samples/show_service_area/show_service_area.dart';
 import 'package:arcgis_maps_sdk_flutter_samples/samples/style_point_with_simple_marker_symbol/style_point_with_simple_marker_symbol.dart';
 
-// A list of all the Widgets for individual Samples.
-// Used by the Sample Viewer App to display the Widget when a sample is selected.
-// The key is the directory name for the sample which is in snake case. E.g. display_map
-final sampleWidgets = <String, Function>{
-  'add_feature_collection_layer_from_table': () =>
+final sampleWidgets = {
+  'AddFeatureCollectionLayerFromTable': () =>
```
Comment (jenmerritt) [inline lib/models/samples_widget_list.dart:47]: This generated file is wrong, no samples are loading in the sample viewer. I'll push a fix to the sample generator.

File: lib/samples/identify_graphics/identify_graphics.png
Comment (jenmerritt) [inline lib/samples/identify_graphics/identify_graphics.png:1]: I think show both the graphic and the alert dialog in the screenshot ideally

File: lib/models/samples_widget_list.dart (hunk 5 above)
Comment (jenmerritt) [reply lib/models/samples_widget_list.dart:47]: #125

File: lib/samples/identify_graphics/identify_graphics.dart [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+        style: SimpleFillSymbolStyle.solid,
+        color: Colors.yellow,
+      ),
+    );
+
+    // Add the graphics to the graphics overlay.
+    _graphicsOverlay.graphics.add(_graphic);
+    // Add the graphics overlay to the map view controller.
+    _mapViewController.graphicsOverlays.add(_graphicsOverlay);
+    // Set the viewpoint to the lake graphic.
```
Comment (PaulAllanSturm) [inline lib/samples/identify_graphics/identify_graphics.dart:96]: How is a "lake" involved?

Comment (shokhinaToj) [reply lib/samples/identify_graphics/identify_graphics.dart:96]: oops sorry I think when I was copying over some stuff from other sample it came along

File: lib/samples/identify_graphics/identify_graphics.png
Comment (shokhinaToj) [reply lib/samples/identify_graphics/identify_graphics.png:1]: Okay I can do that!

File: lib/models/samples_widget_list.dart (hunk 5 above)
Comment (shokhinaToj) [reply lib/models/samples_widget_list.dart:47]: I will fix it and sync this file with v.next!

**Review by PaulAllanSturm — APPROVED** [review]

**Pull request merged**